lib/yellow-sdk/client.rb

```diff
@@ -74,6 +80,8 @@ def make_request(method, endpoint, body)
             request.add_field('API-Key', @api_key)
             request.add_field('API-Nonce', nonce)
             request.add_field('API-Sign', signature)
+            request.add_field('API-Platform', "#{RUBY_PLATFORM} - Ruby #{RUBY_VERSION}")
```

Comment:
RUBY_PLATFORM constant should return the OS release when applicable. So it's a little different than python.